lib/errors/validation-error.ts

```diff
@@ -161,13 +192,13 @@ class ValidationErrorItem {
    *
    * @private
    */
-  getValidatorKey(useTypeAsNS, NSSeparator) {
+  getValidatorKey(useTypeAsNS: boolean, NSSeparator: string): string {
```

Comment:
please rm the `@private` in the jsdoc and specify `private` so TS will pick it up.

```suggestion
  private getValidatorKey(useTypeAsNS: boolean, NSSeparator: string): string {
```

Reply:
This actually is a public function, I believe the `@private` flag was only set because it should be used internally within sequelize and not outside.

I will remove the `@private` flag and keep the function public, I believe this to be the best way. Otherwise, we would have to call `validationErrorItem['getValidatorKey'](...)` instead of `validationErrorItem.getValidatorKey(...)` internally to access the private property